pytube/__main__.py

```diff
@@ -160,8 +160,9 @@ def prefetch(self):
         self.watch_html = request.get(url=self.watch_url)
         if '<img class="icon meh" src="/yts/img' not in self.watch_html:
             raise VideoUnavailable('This video is unavailable.')
```

Comment:
Also, I don't quite understand the logic of this `if` statement. Maybe should it be inverted?